examples/assistant.v1.js

```diff
@@ -6,25 +6,25 @@ var AssistantV1 = require('ibm-watson/assistant/v1');
  * Instantiate the Watson Assistant Service
  */
 var assistant = new AssistantV1({
-  username: process.env.ASSISTANT_USERNAME || '<assistant_username>',
-  password: process.env.ASSISTANT_PASSWORD || '<assistant_password>',
-  version: '2018-02-16'
+  // See: https://github.com/watson-developer-cloud/node-sdk#authentication
+  // iam_apikey: 'INSERT YOUR IAM API KEY HERE',
```

Comment:
I think for the latest version, this should be `authenticator` and not `iam_apikey`

Reply:
does that mean that we need to add an authenticator object and then the apikey?

Reply:
does this mean we need to add an authentication field?